docs/java_cli_for_managedgateway.rst

```diff
@@ -145,8 +145,8 @@ The gateway can invoke sendDeviceManageRequest() method to make the attached dev
 
 .. code:: java
 
-	managedGateway.sendGatewayManageRequest(typeId, deviceId, lifetime, true, true);
-	
+	sendDeviceManageRequest(typeId, deviceId, lifetime, true, true);
```

Comment:
@sathipal  
Also, the next paragraph after the code sample suggests that "lifetime, true, true) are not required in this particular code sample. Could you please clarify? Thanks